cogs/logging.py

```diff
@@ -11,7 +11,16 @@
 class Logging(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
-        self.logging_channel = env.get_logging_channel_name()
+
+        # Set up logging_channels as a dictionary for {guild:channel ID} pairs
```

Comment:
For this to be IDs we'd need to do `channel.id` then, atm it's channels directly which actually is quite convenient though, so probably just fix the comment to `{ guild_id: channel }`.

Reply:
Done.